src/node/mod.rs

```diff
@@ -2,6 +2,7 @@
 pub mod node;
 pub mod operator_event;
 pub mod operator_executor;
+pub mod lattice;
```

Comment:
Since I don't think this is used outside of this module:
```suggestion
pub(self) mod lattice;
```

Reply:
Hmmm, this makes the doctests in `lattice.rs` fail because lattice is no longer visible from `erdos::`.

Reply:
`pub(crate)` then?